test/src/analyzers/lint_analyzer/rules/rules_list/prefer_first_or_null/examples/example.dart

```diff
@@ -0,0 +1,171 @@
+// ignore_for_file: unnecessary_cast, unused_local_variable
```

Comment:
please add desired result,  to show that it does not generate issue with `firstOrNull`